compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -71,7 +71,7 @@ macro_rules! throw_inval {
 
 #[macro_export]
 macro_rules! throw_ub {
-    ($($tt:tt)*) => { Err::<!, _>(err_ub!($($tt)*))? };
+    ($($tt:tt)*) => { Err::<!, _>($crate::err_ub!($($tt)*))? };
```

Comment:
Why does this one macro need fixing but none of the others?

Reply:
because it's the only macro that I used in a place where they needed that. I'll update all of them.